app/controllers/Checkout.scala

```diff
@@ -37,6 +36,22 @@ import scalaz.{NonEmptyList, OptionT}
 
 object Checkout extends Controller with LazyLogging with CatalogProvider {
 
+  val ukAndIsleOfMan = CountryGroup.UK.copy(countries = List(Country.UK, Country("IM", "Isle of Man")))
+  val weeklyUkCountries = CountryGroup.UK.copy(
+    countries = List(
+      Country.UK,
+      Country("GG", "Guernsey"),
+      Country("IM", "Isle of Man"),
+      Country("JE", "Jersey")
+    ))
+
+  val weeklyUkNorthAmericaGroups = List (weeklyUkCountries, CountryGroup.US, CountryGroup.Canada)
+  val weeklyRowGroups = {
+    val rowUk = CountryGroup("Row Uk", "uk", None, CountryGroup.UK.countries.filterNot(weeklyUkCountries.countries.contains(_)), GBP, PostCode)
```

Comment:
this "rowUk" is just an intermediate country group I defined for the Countries in the Uk CountryGroup that are not included in the Uk delivery zone so they use pounds